seisbench/models/base.py

```diff
@@ -486,7 +486,10 @@ def load(cls, path, version_str=None):
         with open(path_json, "r") as f:
             weights_metadata = json.load(f)
         # Load model weights
-        model_weights = torch.load(f"{path_pt}")
+        #MB# : double check the right device for the machine
+        _dev = torch.device("cuda" if torch.cuda.is_available() else "cpu")
```

Comment:
Thanks for the fix. I'd suggest a small change, in that I think by default we should load models always to CPU. Users can still move them to GPU manually afterwards. As our models are small, this doesn't have any performance downsides and it will keep the GPU usage explicit.